(PA-5326) Added platform for macos13-x86_64

amitkarsale opened the pull request:
Added platform for macos13 intel.

mhashizume commented:
Would you be able to rebase on main add this change to the CHANGELOG.md please?

mhashizume commented:
Hi @amitkarsale , have you gotten a chance to take a look at my review? Thank you

amitkarsale commented:
> Hi @amitkarsale , have you gotten a chance to take a look at my review? Thank you

Sorry, I was waiting for other changes to be addressed. I have addressed your comment. Thanks @mhashizume !

mhashizume commented on CHANGELOG.md:
```diff
@@ -6,6 +6,9 @@ This changelog adheres to [Keep a CHANGELOG](https://keepachangelog.com).
+## Added
+-(PA-5326) Added macos 13 (x86_64)
+
 ### Added
```
Can you append your change to the "Added" list that has already been started instead of creating a new "Added" list? Thanks
